specification/cosmos-db/data-plane/Microsoft.Tables/preview/2019-02-02/table.json

```diff
@@ -288,6 +288,99 @@
             }
           }
         }
+      },
+      "get": {
+        "tags": [
+          "table"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "operationId": "Table_Get",
+        "x-ms-examples": {
+          "TableDelete": {
+            "$ref": "./examples/TableGet.json"
+          }
+        },
+        "description": "Operation gets the specified table.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ClientRequestId"
+          },
+          {
+            "$ref": "#/parameters/tableNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "headers": {
+              "x-ms-client-request-id": {
+                "x-ms-client-name": "ClientRequestId",
+                "type": "string",
+                "description": "If a client request id header is sent in the request, this header will be present in the response with the same value."
+              },
+              "x-ms-request-id": {
+                "x-ms-client-name": "RequestId",
+                "type": "string",
+                "description": "This header uniquely identifies the request that was made and can be used for troubleshooting the request."
+              },
+              "x-ms-version": {
+                "x-ms-client-name": "Version",
+                "type": "string",
+                "description": "Indicates the version of the Table service used to execute the request. This header is returned for requests made against version 2009-09-19 and above."
+              },
+              "Date": {
+                "type": "string",
+                "format": "date-time-rfc1123",
+                "description": "UTC date/time value generated by the service that indicates the time at which the response was initiated."
+              }
+            }
+          },
+          "404": {
```

Comment:
Was this included so that 404 isn't treated as an error?

Reply:
Yes - The use case is for an `Exists()` method for a specific table.